### DISCUSSION TITLE
crop images using lximage

### DISCUSSION THREAD

**Discussion by amitoria** [lxqt/lxqt#2005] [Ideas]
Lximage is really easy and simple image viewer. But it would be great, if we can crop image files, using it... It is very common action, performed on images...

**Comment by tsujan** [comment]
It's LXImage-Qt (lximage — the gtk app— isn't developed here).

LXImage-Qt is an image viewer, not an image editor. We definitely don't want to add more image editing features to it, regardless of how common they may be. The current editing functionality (annotations) is suboptimal but, instead of removing it, we hid it by default.
